### DISCUSSION TITLE
Is it possible and/or recommended to add URLs to the classpath of a REPL?

### DISCUSSION THREAD

**Discussion by kincsescsaba** [scala/scala3#21388] [Compiler internals Q&A]
Is it possible and/or recommended to add URLs to the classpath of a REPL using the `-classpath` setting?
I want to create a REPL that loads specific classes via the network.

**Comment by SethTisue** [comment]
This sort of thing is better suited to https://users.scala-lang.org, doesn't seem like a compiler internals question to me

**Reply by kincsescsaba** [reply]
I can partially agree, as my current workaround is implementing a custom REPL with a custom classloader to handle this, which is compiler internals. However, I will try in the user group as suggested.
